sonarts-sq-plugin/its/plugin/src/test/java/org/sonar/typescript/its/TslintRulesTest.java

```diff
@@ -54,6 +54,12 @@ public void S4328_should_consider_peer_dependencies() {
     assertThat(issueList).extracting("line").doesNotContainAnyElementsOf(ImmutableList.of(1));
   }
 
+  @Test
+  public void S4328_should_consider_whitelisted_modules() {
+    List<Issue> issueList = getIssues("S4328", "profile-no-implicit-dependencies-whitelist");
+    assertThat(issueList).extracting("line").doesNotContainAnyElementsOf(ImmutableList.of(2));
```

Comment:
Still not sure to understand. don't we want to be sure to not raise any issue instead? why not adding an case which definitely should raise an issue, in order to be sure to raise only this one.

If the rule is broken, and not raise any issue anymore (while still passing unit test?), i still would to be sure that it works in ITs. Today, you just know that there is no issue raised.

Reply:
I agree with your reasoning here, I will update this assertion.